charts/seed-bootstrap/templates/prometheus/statefulset.yaml

```diff
@@ -44,7 +44,7 @@ spec:
         #   failureThreshold: 10
```

Comment:
@mvladev can we enable liveness and readiness probes now that we use Prometheus 2.3.1?